system/src/system_task.cpp

```diff
@@ -467,6 +467,26 @@ void manage_ble_prov_mode() {
 #endif
 }
 
+extern "C" void system_isr_task_queue_free_memory(uint8_t *ptrToFree) {
+    struct FreeTask : ISRTaskQueue::Task {
+        uint8_t *arg;
```

Comment:
Could this be simpler if we added some kind of argument variable or `void *` context pointer to the base `ISRTaskQueue::Task` instead? I think some other use cases could take advantage of this

Reply:
Potentially, but I'd leave this as-is for now, as all the other usages of ISR task queue already do the same thing anyway.